lib/rules/require-has-block-helper.js

```diff
@@ -14,14 +16,31 @@ function getErrorMessage(name) {
 module.exports = class RequireHasBlockHelper extends Rule {
   visitor() {
     return {
-      PathExpression(node) {
-        if (TRANSFORMATIONS[node.original]) {
-          this.log({
-            message: getErrorMessage(node.original),
-            line: node.loc && node.loc.start.line,
-            column: node.loc && node.loc.start.column,
-            source: this.sourceForNode(node),
-          });
+      PathExpression(node, path) {
+        if (this.mode === 'fix') {
+          if (TRANSFORMATIONS[node.original]) {
+            let parent = path.parent;
+            if (
+              parent &&
```

Comment:
When is `parent` not truthy? I think it always has to be... 🤔

Reply:
Yep, you're right 👍